src/main/java/org/broadinstitute/hellbender/tools/longreads/graph/AlignedBaseGraphCollection.java

```diff
@@ -0,0 +1,441 @@
+package org.broadinstitute.hellbender.tools.longreads.graph;
+
+import htsjdk.samtools.CigarElement;
+import htsjdk.samtools.CigarOperator;
+import org.broadinstitute.hellbender.exceptions.GATKException;
+import org.broadinstitute.hellbender.tools.walkers.haplotypecaller.graphs.SeqVertex;
+import org.broadinstitute.hellbender.utils.read.GATKRead;
+
+import java.io.File;
+import java.util.*;
+
+public class AlignedBaseGraphCollection {
+
+    //==================================================================================================================
+    // Private Members:
+    private final HashMap<String, AlignedBaseGraph>                                 contigSubGraphMap       = new HashMap<>();
+    private final HashMap<String, TreeMap<GenomicAndInsertionPosition, Set<AlignedBaseVertex>>> contigPositionVertexMap
+            = new HashMap<>();
+
+    private boolean isGraphCollapsed = false;
+
+    //==================================================================================================================
+    // Constructors:
+
+    public AlignedBaseGraphCollection() {}
+
+    //==================================================================================================================
+    // Override Methods:
+
+    //==================================================================================================================
+    // Static Methods:
+
+    //==================================================================================================================
+    // Instance Methods:
+
+    /**
+     * Dumps all graphs in this aligned base graph to disk as DOT files.
+     * The files will be named according to their contigs and the given {@code baseName}.
+     *
+     * @param baseName The common base for the names of all output files.
+     */
+    public void serializeToDotFiles(final String baseName) {
+        for ( final Map.Entry<String, AlignedBaseGraph> entry : contigSubGraphMap.entrySet() ) {
+            // Print the graph to a file.  We don't want to prune anything, so give the pruner a VERY LARGE NUMBER.
+            entry.getValue().printGraph(
+                    new File(baseName + "aligned_base_graph." + entry.getKey() + ".dot"),
+                    Integer.MAX_VALUE );
```

Comment:
Set this value to 0 as that is the default we use elsewhere.

Reply:
OBE